v0/destinations/iterable/transform.js

```diff
@@ -147,7 +147,7 @@ function constructPayloadItem(message, category, destination) {
             el,
             mappingConfig[ConfigCategory.PRODUCT.name]
           );
-          if (element.categories) {
+          if (element.categories && typeof element.categories === "stirng") {
```

Comment:
```suggestion
          if (element.categories && typeof element.categories === "string") {
```
can you please correct the string name and same applies for other changes below